--- GoogleAppsAuthentication.php.dist	2014-02-20 12:39:48.736421989 -0500
+++ GoogleAppsAuthentication.php	2014-02-20 15:01:32.012696628 -0500
@@ -56,7 +56,9 @@
 			$googleAccount = getGoogleAccount('title=' . $_REQUEST["title"]);
  
 			// Get username
-			$username = $googleAccount['email'];
+			// http://www.mediawiki.org/wiki/Extension_talk:GoogleAppsAuthentification
+			// $username = $googleAccount['email'];
+			$username = preg_replace('/@.*/', '', $googleAccount['email']);
  
 			// Get MediaWiki user
 			$u = User::newFromName($username);
